src/main/java/it/gov/pagopa/pu/debtpositions/service/create/debtposition/CreateDebtPositionServiceImpl.java

```diff
@@ -72,4 +90,11 @@ private void generateIuv(DebtPositionDTO debtPositionDTO, Boolean pagopaPayment,
         });
     }
   }
+
+  private void invokeWorkflow(DebtPositionDTO debtPositionDTO, Boolean pagopaPayment, String accessToken) {
+    if (Boolean.TRUE.equals(pagopaPayment) && DEBT_POSITION_ORIGIN_TO_SYNC.contains(debtPositionDTO.getDebtPositionOrigin())) {
```

Comment:
move this if condition into the DebtPositionSyncService (you should also accept the "massive" parameter)
The installmentSynchronize API will reuse it